db/migrate/20220811011809_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] Your data model is looking good, great job! However, it would be better to use `t.text` for the `:bio` instead of `t.string`, when you use `string` it has a 255 characters limit, which is better for usernames, emails, etc. Meanwhile, if you used `text` the limit is 4294967296 (default = 65536), this is better for a long text like a biography.